lib/cluster.js

```diff
@@ -694,8 +696,10 @@ function workerInit() {
     if (!this.isConnected()) {
       process.exit(0);
     } else {
-      send({ act: 'exitedAfterDisconnect' }, () => process.disconnect());
-      process.once('disconnect', () => process.exit(0));
+      const processDisconnect = () => process.disconnect();
```

Comment:
ditto here... @jasnell @addaleax is it really the intention that every instance of 

```
some.thing(() => done())
```

in node be rewritten to

```
const doSomething = () => done()
some.thing(doSomething)
```

That is an underwhelming use of a supposed-to-be-more-concise es6 syntax sugar.